packages/grafana-ui/src/themes/mixins.ts

```diff
@@ -40,16 +40,16 @@ export function mediaUp(breakpoint: string) {
   return `only screen and (min-width: ${breakpoint})`;
 }
 
-export const focusCss = (theme: GrafanaTheme) => `
+export const focusCss = (theme: GrafanaTheme2) => `
```

Comment:
Won't this break any plugin that uses this mixin? 🤔

Reply:
You're right, missed this